tests/tuner/test_auto_gpu_select.py

```diff
@@ -0,0 +1,58 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+import torch
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from tests.base.boring_model import BoringModel
+from pytorch_lightning import Trainer
+from pytorch_lightning.tuner.auto_gpu_select import pick_multiple_gpus
+
+
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+@pytest.mark.skipif(
+    torch.cuda.device_count() < 1, reason="test requires a number of GPU machine greater than 1"
+)
```

Comment:
`torch.cuda.device_count() < 1` wont skip a single GPU machine.
Use:
```suggestion
@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
```